tools/chpl-language-server/Makefile

```diff
@@ -44,12 +44,17 @@ test-chpl-language-server: chpl-language-server chpl-language-server-test-venv
 	$(CHPL_MAKE_HOME)/util/config/run-in-venv-with-python-bindings.bash \
 		python3 -m pytest test/*.py
 
-clean: clean-link clean-link-shim
+clean: clean-link clean-link-shim clean-pycache
 
 cleanall: clean
 
 clobber: clean
 
+clean-pycache:
+	rm -rf $(CHPL_MAKE_HOME)/tools/chpl-language-server/src/__pycache__
+	rm -rf $(CHPL_MAKE_HOME)/tools/chpl-language-server/test/__pycache__
+	rm -rf $(CHPL_MAKE_HOME)/tools/chpl-language-server/test/util/__pycache__
```

Comment:
could you just use something like `find $(CHPL_MAKE_HOME)/tools/chpl-language-server/ -name "__pycache__" -delete`?